core/src/packet_deserializer.rs

```diff
@@ -37,37 +35,53 @@ impl PacketDeserializer {
         }
     }
 
+    pub fn unreceived_batch_count(&self) -> usize {
```

Comment:
Need to look at how this is used - my concern is that this might be used in a misleading way.

Our 4 non-vote threads all have the clones of the same receiving channel. Worried this might double (quad) count our queued non-vote packets in metrics.

Reply:
yeah, i was aware of it... after testing it turned out i should use different metrics: https://github.com/solana-labs/solana/pull/29196/commits/e582b019c0075d64e74e20e32c60af54c3f34740

so the concern is over now. :)